package-lock.json

```diff
@@ -469,312 +472,5 @@
         "node": ">= 6"
       }
     }
-  },
-  "dependencies": {
```

Comment:
dependencies are missing. Expected/intentional?

Reply:
This was automatically done by npm, so I'm assuming expected?

Reply:
Probably! We'll find out if something breaks :D